data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/s3/S3ScanService.java

```diff
@@ -36,7 +42,10 @@ public class S3ScanService {
     private final AcknowledgementSetManager acknowledgementSetManager;
     private final S3ObjectDeleteWorker s3ObjectDeleteWorker;
     private final PluginMetrics pluginMetrics;
-    private ScanObjectWorker scanObjectWorker;
+    //private ScanObjectWorker scanObjectWorker;
+    private final ExecutorService executorService;
+    static final int STANDARD_BACKOFF_MILLIS = 30_000;
+    static final int FAST_BACKOFF_MILLIS = 100;
```

Comment:
This is really fast and doesn't really make sense. Let's just make this configurable at the source and keep the default at 30 seconds. Something like

```
source:
  s3:
   scan:
     backoff_time: "1s" // default to 30 seconds
```